lighthouse-core/lib/lh-error.js

```diff
@@ -312,6 +314,11 @@ const ERRORS = {
     message: UIStrings.pageLoadFailedHung,
     lhrRuntimeError: true,
   },
+  /* Used when the page is non-HTML. */
+  INVALID_DOC_TYPE: {
+    code: 'INVALID_DOC_TYPE',
+    message: UIStrings.docTypeInvalid,
```

Comment:
need to also add `lhrRuntimeError: true` here because we want the error to bubble all the way to the top and show up in the LHR [`runtimeError` entry](https://github.com/GoogleChrome/lighthouse/blob/d9b3caba902a54420b3cab4c128b69e0e0a115c0/types/lhr.d.ts#L56) and exit with a non-zero exit code when running the CLI.

@connorjclark's examples in #9245 show that things *mostly* go wrong but not completely, so we need to be extra loud to get the attention of users doing automated runs of LH that they're accidentally auditing a page they either didn't mean to or have misconfigured.

Since it'll be `lhrRuntimeError: true` (and so can also appear in the LHR that PSI serves), it'll also need to be added as an [error entry in the proto](https://github.com/GoogleChrome/lighthouse/blob/d9b3caba902a54420b3cab4c128b69e0e0a115c0/proto/lighthouse-result.proto#L12).

Reply:
Thanks, I'll make sure to fix this!